src/test/test-models/stanc.cpp

```diff
@@ -1,14 +1,14 @@
 #include <stan/command/stanc_helper.hpp>
 
-/** 
+/**
  * The Stan compiler (stanc).
  *
  * @param argc Number of arguments
  * @param argv Arguments
- * 
+ *
  * @return 0 for success, -1 for exception, -2 for parser failure, -3
  * for invalid arguments.
  */
 int main(int argc, const char* argv[]) {
-  return stanc_helper(argc, argv, &std::cout, &std::cerr);
+  return stanc_helper(argc, argv, NULL, NULL);
```

Comment:
Are you sure these NULL values work everywhere?  Otherwise, the safer thing to do would be to create a local stringstream and pass pointers to those.  That'll also let you run tests on what comes out.